generate/generate.go

```diff
@@ -782,6 +782,20 @@ func (s *service) generateHelperFuncs(a *API) {
 			pn("		return nil, -1, err")
 			pn("	}")
 			pn("")
+			pn("    if l.Count == 0 {")
+			pn("		// look	inside projects ")
+			pn("		p.p[\"projectid\"] = \"-1\"")
+			pn("		l, err := s.List%s(p)", ln)
```

Comment:
You now make shadow variables by using the declare and assign operator `:=`, you should only assign the values here using `=` otherwise the resulting `l` will only exist until the end of this `if` body after which `l` will be the emtpy `l` we already had before...